docker-compose/obam-with-obkm/docker-compose.yml

```diff
@@ -42,8 +42,8 @@ services:
       start_period: 30s
 
   obkm:
-    container_name: obkm
```

Comment:
Better to run a complete flow test when time permits.

Reply:
Tested basic portals such as /publisher, /devportal, /carbon and etc. Will have to do integration testing in future.